src/components/radio/examples/RadioSelectExample.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { Radio } from '../Radio';
+import { RadioSelect } from '../RadioSelect';
+
+export interface IRadioSelectProps {
+  defaultValue?: string;
+  disabled?: boolean;
+}
+
+export interface IRadioSelectState {
+  value: string;
+}
+
+export class RadioSelectExample extends React.Component<IRadioSelectProps, IRadioSelectState> {
+  constructor(props: IRadioSelectProps, state: IRadioSelectState) {
+    super(props, state);
+    this.state = {
+      value: this.props.defaultValue,
+    };
+  }
+
+  handleChange(value: string) {
```

Comment:
Are examples meant to show the usage of a React Component? If so, why is there base behavior directly in the example?

Reply:
Yes, the examples are used to demonstrate the usage of components. We make them dynamics by having a state and modifying it as someone would do it their own application

Also the method should be private